workspaces/src/network/server.rs

```diff
@@ -111,22 +148,20 @@ impl Drop for SandboxServer {
             return;
         }
 
+        let rpc_port = self.rpc_port();
         let child = self.process.as_mut().unwrap();
```

Comment:
I believe this would panic for custom sandbox urls.

Reply:
For custom sandbox URLs, there's no process spawned, so `child` would be None here and early return in the above:
```rust
        if self.process.is_none() {
            return;
        }
```

Reply:
Ah, that code path was collapsed in the diff. Looks good.

Reply:
Wanna try this at the top instead?

```rust
let child = match self.process.take() {
    Some(child) => child,
    None => return,
};
```